template.yaml

```diff
@@ -662,3 +683,7 @@ Outputs:
     Export:
       Name: !Sub "${AWS::StackName}-F2FFrontGatewayId"
     Value: !Ref ApiGwHttpEndpoint
+  F2FCustomDomain:
+    Description: F2F Custom Domain API Gateway ID
+    Export:
+      Name: !Sub "${AWS::StackName}.${DNSSUFFIX}"
```

Comment:
it would be better to reference the created resource to get the output - this was a future change won't actually break it. e.g. `!Ref F2FFrontCustomDomain`